ios/FluentUI/Command Bar/CommandBarTokens.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class MSFCommandBarTokens: MSFTokensBase, ObservableObject {
+    @Published public var defaultBackgroundColor: UIColor!
```

Comment:
Same here for the @Published properties. We don't need these if we're not using it in a SwiftUI View.